scripts/west_commands/sign.py

```diff
@@ -552,6 +566,7 @@ def sign(self, command, build_dir, build_conf, formats):
         is_sof_build = build_conf.getboolean('CONFIG_SOF')
         if not is_sof_build:
             no_manifest = True
+            self.generate_uuid_registry()
```

Comment:
FWIW I don't think making non-SOF Zephyr builds look like SOF is a very good idea...